CS361Project/urls.py

```diff
@@ -0,0 +1,30 @@
+"""CS361Project URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.contrib import admin
+from django.urls import path
+from TAScheduler.views import Login, Home, CreateUser, CreateCourse, AccountSettings, EditUser, EditCourse
```

Comment:
```suggestion
from TAScheduler.views import *
```
The asterisk imports all the classes from views so they don't have to be all typed out.